_posts/2017-02-06-the-4-key-elements-of-a-conversation.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: post
+asset-type: post
+name: The 4 key elements of a conversation
+title: 'The 4 key elements of a conversation'
+date: 2017-02-08 09:00:00 +00:00
+author: Giulia Mantuano
+image:
+  src: /assets/img/custom/blog/2017-02-06-conversational-interface-design-fundamentals.jpg
+tags:
+- UX
+- Chatbot
+- Conversation Design
+- Conversational Interfaces
+
+---
+Conversational Interfaces is a term often used to define systems that hold "human-like" conversations, examples include; Chatbots, Voice User Interfaces, Embodied Conversational Agents. The user interacts with these using text input, voice recognition, and gestures. These types of interfaces are often unable to provide a good conversational experience. In order to create a better experience we need to be aware of the following 4 key elements of human-to-human conversations.
```

Comment:
"Conversational Interfaces" is a term often used to define systems that hold "human-like" conversations, such as Chatbots, Voice User Interfaces or Embodied Conversational Agents. The user interacts with these using text input, voice recognition, and gestures. These types of interfaces are often unable to provide a good conversational experience. In order to create a better experience we need to be aware of the following four key elements of human-to-human conversations.